frontend/src/components/Gym/ConfirmationModal.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react'
+import Button from '../Button'
+import styled from 'styled-components'
+
+import 'antd/dist/antd.css'
```

Comment:
is this necessary? not sure if the global styles file already imported it, do check!

Reply:
doesn't look like it's in the global styles for rhapp tho